EPP_namespaces.txt

```diff
@@ -0,0 +1,148 @@
+This is the list of all EPP XML namespaces supported by this library,
```

Comment:
Only includes the namespaces from original PM Net-DRI-0.96_10 version correct?

Reply:
yes, will update for future versions and check our namespaces when all is complete.